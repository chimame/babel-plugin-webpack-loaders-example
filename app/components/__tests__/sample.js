import sample from '../sample'

describe('toMatchSnapshot example', () => {
  test('render sample', () => {
    expect(sample()).toMatchSnapshot()
  })
})
