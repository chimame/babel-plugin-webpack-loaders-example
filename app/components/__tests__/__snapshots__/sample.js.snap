exports[`toMatchSnapshot example render sample 1`] = `
<ul
  className="_2LiZgG84CcVl9AXOa1wZdC">
  <li>
    first element
  </li>
  <li>
    second element
  </li>
  <li>
    third
  </li>
</ul>
`;
